lib/build/clang

```diff
@@ -11,7 +11,7 @@ for x in $p
 do
     case "$x" in
         -o) set -- "$@" -o ; extension=".bc" ;;
-        -c) xlinker= ;&
+        -c) xlinker= ; set -- "$@" "$x$extension" ; extension= ;;
```

Comment:
@iscottb122 can you comment on whether or not this is correct? The reason for the change is that `;&` is a bashism and isn't POSIX compliant apparently. My understanding is that it's just doing a fall through to the default case. So I've just copied the code as a 'quick fix'. There's probably a better way though.